examples/ct/cert_checker.go

```diff
@@ -107,7 +108,7 @@ func ValidateChain(rawChain [][]byte, trustedRoots PEMCertPool) ([]*x509.Certifi
 
 		for i, certInChain := range chainMinusRoot {
 			if certInChain != chain[i] {
-				continue
+				continue nextVerifiedChain
```

Comment:
Could this be restructured to avoid the continue by returning true when it succeeds and falling through to return false when it fails.